### DISCUSSION TITLE
Batch.from_data_list inconsistency

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#4601] [General]
Batch.from_data_list is not working for this graph
![image](https://user-images.githubusercontent.com/51344466/167246202-0e542b6a-1842-4cb6-91bc-2a8bab2da606.png)
![image](https://user-images.githubusercontent.com/51344466/167246224-6091df63-6fb6-4ecf-81d5-c6e2e6bdb56f.png)
But if the first edge is not empty like this:
![image](https://user-images.githubusercontent.com/51344466/167246271-94688fe5-edba-48c7-b27e-c3a805ea2e83.png)

it works

**Comment by rusty1s** [comment]
Thanks for reporting. Do you have a small script to reproduce? This would help me to debug this more easily.

**Reply by DogitoErgoSum** [reply]
The problem was in my model metadata, sorry for the time lost @rusty1s
